src/Library/demos/List View/main.js

```diff
@@ -0,0 +1,92 @@
+import Gio from "gi://Gio";
+import Gdk from "gi://Gdk";
+import GObject from "gi://GObject";
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+
+const stack = workbench.builder.get_object("stack");
+const list_view = workbench.builder.get_object("list_view");
+const grid_view = workbench.builder.get_object("grid_view");
+const add = workbench.builder.get_object("add");
+const remove = workbench.builder.get_object("remove");
+const model_for_listview = workbench.builder.get_object("model_for_listview");
+
+//Model
+let item = 1;
+const string_model = new Gtk.StringList({
+  strings: ["Default Item 1", "Default Item 2", "Default Item 3"],
+});
+
+const model = new Gtk.SingleSelection({ model: string_model });
+
+const factory_for_grid_view = new Gtk.SignalListItemFactory();
+factory_for_grid_view.connect("setup", (factory, listItem) => {
+  const listBox = new Gtk.Box({
+    width_request: 160,
+    height_request: 160,
+    css_classes: ["card"],
+  });
+  const label = new Gtk.Label({
+    halign: Gtk.Align.CENTER,
+    hexpand: true,
+    valign: Gtk.Align.CENTER,
+  });
+  listBox.append(label);
+  listItem.set_child(listBox);
+});
+factory_for_grid_view.connect("bind", (factory, listItem) => {
+  const listBox = listItem.get_child();
+  const modelItem = listItem.get_item();
+  const labelWidget = listBox.get_last_child();
+
+  labelWidget.label = modelItem.string;
+});
+
+/*model_for_grid_view.model.connect(
+  "items-changed",
+  (list, position, removed, added) => {
+    console.log(
+      `position: ${position}, Item removed? ${Boolean(
+        removed,
+      )}, Item added? ${Boolean(added)}`,
+    );
+  },
+);*/
+
+model.connect("selection-changed", () => {
+  const list_view_selected_item = model.get_selected();
+  const grid_view_selected_item = model.get_selected();
+  console.log(
+    `Selected item from ListView: ${model.model.get_string(
+      list_view_selected_item,
+    )}`,
+  );
+  console.log(
+    `Selected item from GridView: ${model.model.get_string(
+      grid_view_selected_item,
+    )}`,
+  );
```

Comment:
Since we're operating on a single list model, let's trim this to one log message and try to make that more obvious.

```suggestion
  const selected_item = model.get_selected();
  console.log(
    `Model item selected from view: ${model.model.get_string(
      selected_item,
    )}`,
  );
```